lib/fs/storage.go

```diff
@@ -35,6 +35,23 @@ type Storage struct {
 	vol string
 }
 
+// List reads the path content
+func (s Storage) List(path string) ([]string, error) {
+	var err error
+
+	dir, err := s.fullPath(path)
+	if err != nil {
+		return []string{}, err
+	}
+	d, err := os.Open(dir)
+	if err != nil {
+		return []string{}, err
+	}
+	defer d.Close()
+	names, err := d.Readdirnames(-1)
```

Comment:
We can just return this, no need to assign results to variables and then return.